src/components/custom-field-input-text/custom-field-input-text.css

```diff
@@ -0,0 +1,84 @@
+@import '../../styles/colors.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography.css';
+
+.additional-information-icon {
+  border: 1px solid var(--palette-grey-light);
+  border-radius: 50%;
+  color: var(--palette-grey-dark);
+  cursor: pointer;
+  display: inline-block;
+  height: var(--spacing-large);
+  left: var(--spacing-medium);
+  padding-left: 1px;
```

Comment:
It seems this is causing the circle to be more oval-y than a value of `0px`:

1px 

![Screen Shot 2019-12-05 at 5 32 04 PM](https://user-images.githubusercontent.com/841084/70287828-2f4ebd00-1785-11ea-865f-ffafb5e32891.png)

0px

![Screen Shot 2019-12-05 at 5 32 26 PM](https://user-images.githubusercontent.com/841084/70287845-3aa1e880-1785-11ea-9c47-65b094a40eac.png)